app/models/pool.rb

```diff
@@ -23,15 +23,28 @@ class Pool < ActiveRecord::Base
 
   DEFAULT_POOL = 'Unassigned'
 
+  def remove_pool
```

Comment:
I feel like this should be in `Tournament` model since we're reaching into the tournament and its other pools... but this is ok.